include/avif/avif.h

```diff
@@ -531,6 +531,18 @@ AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, a
 //   * x/x/[1|2|5|6|9]/Limited
 //   * [1|2|5|6|9]/x/12/Limited
 
+// If libavif is built with libyuv fast paths enabled, libavif will use libyuv for conversion from
+// RGB to YUV if the following requirements are met:
+//
+// * YUV depth: 8
+// * RGB depth: 8
+// * rgb.chromaDownsampling: AVIF_CHROMA_UPSAMPLING_AUTOMATIC, AVIF_CHROMA_UPSAMPLING_FASTEST
```

Comment:
Also image->matrixCoefficients must be `AVIF_MATRIX_COEFFICIENTS_BT470BG` or `AVIF_MATRIX_COEFFICIENTS_BT601`.  We probably should use `5` and `6`, following the convention used at lines 529-532.